packages/shared/clickhouse/migrations/0004_scores_add_queue_id.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scores
+ADD COLUMN queue_id Nullable(String) AFTER string_value;
```

Comment:
@Steffen911 FYI, i assume this is missing in the migration as well?

Reply:
When a user creates a score from the Annotation queue, we add this to the score upon score creation. I would add it here for sure.

Reply:
@maxdeichmann I included this change in https://github.com/langfuse/langfuse/pull/3960/files#diff-ac64ef43424291171fd4ebfbd3fbaca280f563f6ddce31e08b834586804dea34R15. Hence you can remove it here.